packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -18,13 +18,13 @@ Source0:    https://downloads.theforeman.org/%{name}/%{name}-%{version}%{?prerel
 BuildArch:  noarch
 
 Requires:   curl
-Requires:   puppet-agent >= 5.5.0
+Requires:   puppet-agent >= 6.15.0
```

Comment:
I've now unified this to 6.15.0 across the board given puppet-candlepin needs 6.15.0 as a minimum due to enabling the pki-core module.